lib/fencing/st_output.c

```diff
@@ -481,7 +481,7 @@ stonith_event_xml(pcmk__output_t *out, va_list args)
         default: {
             char *state = pcmk__itoa(event->state);
             pcmk__xe_set_props(node,
-                               PCMK_XA_STATUS, "pending",
+                               PCMK_XA_STATUS, PCMK_VALUE_PENDING,
```

Comment:
a good example of https://github.com/ClusterLabs/pacemaker/pull/3308#issuecomment-1891069059 -- not an option value, just "something with special meaning that's used as the the value of an XML attribute."

Reply:
Okay sounds good, that's how I'd prefer to use PCMK_VALUE. We started out using them for enumerated values, so I wondered if we wanted to branch off other conventions for things that weren't enumerated values.